lib/controller.js

```diff
@@ -274,11 +275,6 @@ Tessel.get = function(opts) {
       if (opts.name && opts.name === tessel.name) {
         finishSearchEarly(tessel);
       }
-      // If we just found a USB connection and should prefer it
-      else if (!opts.name && tessel.usbConnection !== undefined && !opts.lanPrefer) {
-        // Finish early with this Tessel
-        finishSearchEarly(tessel);
-      }
```

Comment:
Shouldn't this condition should stay intact, only skipping `finishSearchEarly(tessel);` if there are more than one attached?